core-developers/become-core-developer.rst

```diff
@@ -38,10 +38,12 @@ are granted through these steps:
 #. If the candidate receives at least two-thirds positive votes when the poll closes
    (as per :pep:`13`), the submitter `emails the steering council
    <mailto:steering-council@python.org>`_ with the candidate's email address
-   requesting that the council either accept or reject the proposed membership.
+   requesting that the council either accept or reject the proposed membership.  Technically, the
+   council may only `veto a positive vote <https://peps.python.org/pep-0013/#membership>`_.
```

Comment:
Perhaps "requesting that the council confirms that it does not `veto the proposed membership`_."?